addon/templates/components/cfb-form-editor/question-list.hbs

```diff
@@ -40,10 +40,7 @@
       placeholder="{{t "caluma.form-builder.global.search"}}..."
       value={{search}}
       oninput={{action
-        (queue
-          (action (mut search))
-          (action (perform data))
-        )
+        "search"
         value="target.value"
       }}
```

Comment:
Use the `{{on}}` helper

Reply:
I would need to update the component to octane to do that

Reply:
I would need to update the component to octane to do that